api/routers/metrics/metrics.py

```diff
@@ -27,7 +27,7 @@ def validate_camera_existence(camera_id: str):
 
 
 def get_areas(areas: str) -> Iterator[str]:
-    if areas:
+    if areas and areas.upper() != "ALL":
```

Comment:
`areas` is a list of strings. I think that the condition should be
```
if areas and not "ALL" in areas:
```

Reply:
I don't understand this, if the constant is in the list it should be split as normal, right?

Reply:
Changed!